pyiron/atomistics/master/elastic.py

```diff
@@ -0,0 +1,271 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+from __future__ import print_function
+
+import numpy as np
+import scipy.constants
+from pyiron.atomistics.master.parallel import AtomisticParallelMaster
+from pyiron_base import JobGenerator
+from sklearn.linear_model import LinearRegression
+from collections import defaultdict
+
+__author__ = "Joerg Neugebauer, Jan Janssen"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Jan Janssen"
+__email__ = "janssen@mpie.de"
+__status__ = "production"
+__date__ = "Sep 1, 2017"
```

Comment:
We'll probably want to update that to at least this year. ;)

Reply:
oh I didn't realise that it was not 2017 anymore :D